libs/api/src/lib/graphql/payment/payment.private-mutation.ts

```diff
@@ -27,7 +27,7 @@ export const createPaymentFromInvoice = async (
   const invoice = await invoicesByID.load(invoiceID)
   const memberPlan = await memberPlanClient.findFirst({
     where: {
-      Subscription: {
```

Comment:
why did it work so far? this is a crucial functionality imo.

Reply:
I renamed it in the schema.